sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseEditAuthorization.java

```diff
@@ -32,7 +32,7 @@ public Boolean caseEditAllowedCheck(Case caze) {
 
 		User user = userService.getCurrentUser();
        
-        if (DataHelper.equal(user.getUuid(), (caze.getUuid()))) {
+        if (DataHelper.equal(user.getUuid(), (caze.getReportingUser().getUuid()))) {
```

Comment:
To be safe, let's add a null check to the reporting user here. Same in the ContactEditAuthorization

Reply:
done